nixos/tests/vscodium.nix

```diff
@@ -70,15 +70,15 @@ let
 
             # Save the file
             machine.send_key('ctrl-s')
-            machine.wait_for_text('Save')
+            machine.wait_for_text('(Save|Desktop|alice|Size)')
             machine.screenshot('save_window')
             machine.send_key('ret')
 
             # (the default filename is the first line of the file)
             machine.wait_for_file(f'/home/alice/{test_string}')
 
-        machine.send_key('ctrl-q')
-        machine.wait_until_fails('pgrep -x codium')
+        # machine.send_key('ctrl-q')
```

Comment:
Why are these commented? Or why do they fail?

Reply:
Not sure _why_ they fail, but they do... Vscode simply sometimes swallows the `ctrl-q`, which would still cause it to hang

Reply:
> Vscode simply sometimes swallows the `ctrl-q`

probably similar issue https://github.com/keeweb/keeweb/issues/574

Reply:
Ok, one could retry sending the `ctrl-q` a few times?